tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/bigquery_tests_base.py

```diff
@@ -247,6 +251,12 @@ def initialize_class_vars(cls):
             # The query class that is being executed.
             cls.query_class = None
 
+        def setUp(self):
+            """
+            Add data to the tables for the rule to run on.
+            """
+            super().setUp()
```

Comment:
Can you explain the reason why this was added again?

Reply:
Cleaning classes that extend this class may define their own `setUp` function.  If the extending class calls `super().setUp()`, it will be routed to this function.  If this function does not exist, it will instead be routed to `BigQueryTestBase.setUp()`.  Both setups are functional.  Without this function, the hierarchy looks like:
```
class BigQueryTestBase():
    def setUp():

class CleaningRulesTestBase(BigQueryTestBase):

class ExtendingCleaningRuleTest(CleaningRulesTestBase):
    def setUp():
        # calls the setup function in BigQueryTestBase
        super().setUp()  
```

With it, the hierarchy is:
```
class BigQueryTestBase():
    def setUp():

class CleaningRulesTestBase(BigQueryTestBase):
    def setUp():
        # calls the setup function in BigQueryTestBase
        super().setUp()  

class ExtendingCleaningRuleTest(CleaningRulesTestBase):
    def setUp():
        # calls the setup function in CleaningRulesTestBase
        super().setUp()  
```

I added it to the Cleaning Rules Test Base, to give a visual clue to future developers of where to add cleaning rule specific setUp functions.  There may come a time when we want to add something for cleaning rules but not necessarily for all big query setup functions.

Does this  make sense?  Or would a call help?

Reply:
Yes, that makes sense! Thank you!